plugins/withAndroidStylesAccentColorPlugin.js

```diff
@@ -12,7 +12,7 @@ module.exports = function withAndroidStylesAccentColorPlugin(appConfig) {
         decoratedAppConfig.modResults,
         {
           add: true,
-          parent: AndroidConfig.Styles.getAppThemeLightNoActionBarGroup(),
```

Comment:
`getAppThemeLightNoActionBarGroup` is deprecated and will not work on subsequent prebuild without `--clean` as `AppTheme` parent is replaced.